src/com/clover/remote/client/transport/websocket/WebSocketCloverTransport.ts

```diff
@@ -196,13 +196,9 @@ export abstract class WebSocketCloverTransport extends CloverTransport implement
         // Attempt to clear out messages already in the send queue
         this.drainQueue();
 
-        if (this.webSocket != null) {
+        if (this.cloverWebSocketClient != null) {
             this.notifyDeviceDisconnected();
-            try {
-                this.webSocket.close();
-            } catch (e) {
-                this.logger.error('error disposing of transport.', e);
-            }
+            this.cloverWebSocketClient.close();
```

Comment:
The try/catch was moved to CloverWebSocketInterface this seems to make more sense.